### PR TITLE
Payments: Rename `fees` to `share`

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: docs/4-entry-points/6-payments-book.md [hunk 1]
```diff
@@ -46,42 +46,42 @@ The payments module does not enforce users of Aragon Protocol to pay on-chain. T
...
 ### 4.6.5. Ensure period balance details
 
-- **Actor:** External entity incentivized in updating the parameters to determine the guardians share fees for each period
+- **Actor:** External entity incentivized in updating the parameters to determine the guardians share for each period
 - **Inputs:**
     - **Period ID:** Period identification number
 - **Authentication:** Open
 - **Pre-flight checks:**
-    - Ensure that all the terms corresponding to the requested period were already been initialized for the Protocol
+    - Ensure that all the terms corresponding of the requested period have already been initialized for the Protocol
```
Comment (sohkai) [inline docs/4-entry-points/6-payments-book.md:76]: What does "initialized" mean here? That they've all started?

File: packages/evm/test/helpers/utils/errors.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ const PAYMENTS_BOOK_ERRORS = {
   ETH_TRANSFER_FAILED: 'PB_ETH_TRANSFER_FAILED',
   TOKEN_DEPOSIT_FAILED: 'PB_TOKEN_DEPOSIT_FAILED',
   TOKEN_TRANSFER_FAILED: 'PB_TOKEN_TRANSFER_FAILED',
-  GUARDIAN_FEES_ALREADY_CLAIMED: 'PB_GUARDIAN_FEES_ALREADY_CLAIMED',
+  GUARDIAN_SHARE_ALREADY_CLAIMED: 'PB_GUARDIAN_SHARE_ALREADY_CLAIMED',
```
Comment (sohkai) [inline packages/evm/test/helpers/utils/errors.js:79]: I've seen a couple of errors now that are over 32 chars long; would it be worth creating a quick test to go through these objects and count each string's length?

Or do we not care too much about this at the moment?

**Review by sohkai — COMMENTED** [review]
LGTM, just left a couple of changes that I thought may help clarify the terminology in https://github.com/aragon/protocol/pull/17.

File: docs/4-entry-points/6-payments-book.md (hunk 1 above)
Comment (facuspagnuolo) [reply docs/4-entry-points/6-payments-book.md:76]: Yes. Fixed in 82aea7a3b78501a84d5351f3fce3c932ca5b1f63

**Pull request merged**